cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -343,10 +343,14 @@ public void testPvcNames() {
         KafkaCluster kc = KafkaCluster.fromCrd(assembly, VERSIONS);
 
         List<PersistentVolumeClaim> pvcs = kc.getVolumeClaims();
-
-        for (int i = 0; i < replicas; i++) {
-            assertEquals(kc.VOLUME_NAME + "-" + KafkaCluster.kafkaPodName(cluster, i),
-                    pvcs.get(0).getMetadata().getName() + "-" + KafkaCluster.kafkaPodName(cluster, i));
+        List<VolumeMount> vms = kc.getVolumeMounts();
+        for (int i = 0; i < pvcs.size(); ++i) {
+            String name = pvcs.get(i).getMetadata().getName();
+            assertEquals(kc.VOLUME_NAME, name);
+            VolumeMount vm = vms.get(i);
+            assertEquals(name, vm.getName());
+            assertEquals("/var/lib/kafka/" + name, vm.getMountPath());
+            assertNull(vm.getSubPath());
```

Comment:
That is a valid test. But we should probably also have some test with the `subPath` set. I think it should be also either tested in `ZookeeperClusterTests` or done one level above in `StatefulClusterTests`.